lib/upgrade_js.js

```diff
@@ -218,6 +219,25 @@ class Script {
       declarationNames.add(getDeclName(decl));
       declaration.properties.push(decl);
     });
+
+    if (elementInfo.polyfillTokenList) {
+      const ast = espree.parse(`
+        (function (obj) {
+          var pieces = [];
+          for (key in obj) {
```

Comment:
Did tokenList run on prototypical properties as well? If not, we should use `Object.keys`.